src/main/java/appeng/client/gui/AEBaseGui.java

```diff
@@ -84,7 +84,7 @@
 
 public abstract class AEBaseGui extends GuiContainer
 {
-	private final List<InternalSlotME> meSlots = new LinkedList<>();
+	private final List<InternalSlotME> meSlots = new ArrayList<>();
 	// drag y
 	private final Set<Slot> drag_click = new HashSet<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "drag_click" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00116)